### PR TITLE
Constitution updates  

### PR DESCRIPTION
**Pull request by YaserJazouane**
* reflect counsel review
* include the list of wallets and SCs used by governance

### CODE REVIEW

File: docs/constitution/pyth-dao-constitution.md [hunk 1]
```diff
@@ -2,46 +2,45 @@
...
-- **Pyth Improvement Proposal (PIP)**
-- **Pyth DAO Treasury:** all tokens held in the governance smart contract that is directly governed by the Pyth DAO via on-chain voting
-- **Pyth Forum:** the system adopted by the DAO to manage PIP, organize the voting process and provide connectivity to the governance contract on the Solana Blockchain
-- **Pythian Multisig Wallet:** smart contract wallet signed by the elected members of the Pythian Council
-- **Price Feed Multisig Wallet:** smart contract wallet signed by the elected members of the Price Feed Council
-- **Operations Wallet:** wallet used by council members to submit PIPs on-chain. Each council has access to a dedicated and separate Operations Wallet. This shared wallet can be used by automated processes to make proposals or by council members to propose large proposals that would otherwise require many hardware wallet signatures.
-- **PGAS:** utility token exclusively used in the Pythnet Appchain, governed by the Pyth DAO
+- **Votable Token:** PYTH tokens staked token in the governance contract at the following address: pytGY6tWRgGinSCvRLnSv4fHfBTMoiDGiCsesmHWM6U
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md:5]: No they're staked in the staking contract `pytS9TjG1qyAZypk7n8rw8gfW9sUaqqYyMhJQ4E7JCQ`

File: docs/constitution/pyth-dao-constitution.md [hunk 2]
```diff
@@ -2,46 +2,45 @@
...
+- **PIP:** Pyth Improvement Proposal
+- **Pyth DAO Treasury:** all tokens held in the governance smart contract, at the following address: [treasury_wallet_address] , that is directly governed by the Pyth DAO via on-chain voting
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md]: Gx4MBPb1vqZLJajZmsKLg8fGw9ErhoKsR8LeKcCKFyak for SOL treasury
9HKkxg5dpqjUEW1U2r76SpQCH7uvDMciytNYxrpwMVNc for SPL tokens

File: docs/constitution/pyth-dao-constitution.md (hunk 1 above)
Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: fixed

File: docs/constitution/pyth-dao-constitution.md [hunk 3]
```diff
@@ -2,46 +2,45 @@
...
+- **Pyth Forum:** the system adopted by the Pyth DAO to manage PIPs, organize the voting process and provide connectivity to the governance contract on the Solana Blockchain, located here: [URL]
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md]: Should we add the Discord url

File: docs/constitution/pyth-dao-constitution.md [hunk 4]
```diff
@@ -2,46 +2,45 @@
...
+- **Pythian Multisig Wallet:** smart contract wallet, at the following address 6oXTdojyfDS8m5VtTaYB9xRCxpKGSvKJFndLUPV3V3wT, signed by the elected members of the Pythian Council. The Pythian Council is described in further detail below.
+- **Price Feed Multisig Wallet:** smart contract wallet, at the following address CPUZ9njbfxjqrsx713gEk8KYatvoz1Y4hvs4a5UQKEiu signed by the elected members of the Price Feed Council. The Price Feed Council is described in further detail below.
+- **Operations Wallets:** wallets, at the following addresses opsLibxVY7Vz5eYMmSfX8cLFCFVYTtH6fr6MiifMpA7 and ACzP6RC98vcBk9oTeAwcH1o5HJvtBzU59b5nqdwc7Cxy for the Pythian Council and the Price Feed Council respectively, used by council members to submit PIPs on-chain. Each council has access to a dedicated and separate Operations Wallet. This shared wallet can be used by automated processes to make proposals or by council members to propose large proposals that would otherwise require many hardware wallet signatures.
+- **PGAS:** utility token, at the following address DgpbK8SiypiUHBkBTAunMnwRWF3McGGR4iKxTrTfTXq4 exclusively used in the Pythnet Appchain, governed by the Pyth DAO
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md]: This isn't accurate, PGAS is the native token of Pythnet, it doesn't have an address

File: docs/constitution/pyth-dao-constitution.md [hunk 5]
```diff
@@ -2,46 +2,45 @@
...
+The Pyth DAO LLC, is legally structured as “a non-profit DAO LLC formed under the laws of the Republic of Marshall Islands, formed to serve the Pyth DAO LLC” (of which the OPERATING AGREEMENT OF PYTH DAO LLC is available at: [ipfs://QmP2GmL1n2WbHd7AtHqyXVWFyyHH36aZLfVZbNoqhommJi](https://cloudflare-ipfs.com/ipfs/QmP2GmL1n2WbHd7AtHqyXVWFyyHH36aZLfVZbNoqhommJi)). The wrapperPyth DAO is algorithmically managed, such that actions taken by the Pyth DAO via the governance contract on the Solana Blockchain are deemed to be actions of the Pyth DAO LLC. The legal entity enables the Pyth DAO LLC to hold the treasury, protect Pyth DAO members from unlimited liability, and allow Pyth DAO members to take part in governance by providing a clear framework in respect of the rights and duties of Pyth tokenholders that hold Votable Tokens.
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md:21]: `wrapperPyth` missing space

File: docs/constitution/pyth-dao-constitution.md (hunk 3 above)
Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: added

File: docs/constitution/pyth-dao-constitution.md (hunk 4 above)
Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: fixed

File: docs/constitution/pyth-dao-constitution.md (hunk 5 above)
Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: fixed

File: docs/constitution/pyth-dao-constitution.md [hunk 6]
```diff
@@ -101,14 +100,15 @@ Any Council member who has voted less than 1/3 of the proposals during the term
 
 The following process governs the election that starts at time T:
 
-- Nomination (T until T+2 days): Any DAO member can nominate himself or herself for candidacy to the Council. Each candidate sponsor, a person nominating a candidate or the candidate himself or herself, must hold at least 50,000 of Votable Tokens
-- Member Election (T+2 days until T+7 days): Each token may be cast for one candidate.
+- Nomination (T until T+2 days): Any Pyth DAO LLC member can nominate himself or herself for candidacy to the Council. Each; provided that each candidate sponsor, a person nominating a candidate or the candidate himself or herself, must hold at least 50,000 of Votable Tokens
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md:103]: > Each; provided that each candidate sponsor, a person nominating a candidate or the candidate himself or herself, must hold at least 50,000 of Votable Tokens 

This sentence sound weird

File: docs/constitution/pyth-dao-constitution.md [hunk 7]
```diff
@@ -2,46 +2,45 @@
...
-- **Pythian Multisig Wallet:** smart contract wallet signed by the elected members of the Pythian Council
-- **Price Feed Multisig Wallet:** smart contract wallet signed by the elected members of the Price Feed Council
-- **Operations Wallet:** wallet used by council members to submit PIPs on-chain. Each council has access to a dedicated and separate Operations Wallet. This shared wallet can be used by automated processes to make proposals or by council members to propose large proposals that would otherwise require many hardware wallet signatures.
-- **PGAS:** utility token exclusively used in the Pythnet Appchain, governed by the Pyth DAO
+- **Votable Token:** PYTH tokens staked token in the staking contract at the following address: `pytS9TjG1qyAZypk7n8rw8gfW9sUaqqYyMhJQ4E7JCQ`
+- **PIP:** Pyth Improvement Proposal
+- **Pyth DAO Treasury:** all tokens held in the governance smart contract, at the following address: `Gx4MBPb1vqZLJajZmsKLg8fGw9ErhoKsR8LeKcCKFyak` , that is directly governed by the Pyth DAO via on-chain voting
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md:7]: Can you add 9HKkxg5dpqjUEW1U2r76SpQCH7uvDMciytNYxrpwMVNc for the SPL token treasury.
Gx4MBPb1vqZLJajZmsKLg8fGw9ErhoKsR8LeKcCKFyak is only for native SOL.

Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: changed

File: docs/constitution/pyth-dao-constitution.md (hunk 6 above)
Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: fixed

File: docs/constitution/pyth-dao-constitution.md (hunk 5 above)
Comment (jayantk) [inline docs/constitution/pyth-dao-constitution.md]: typo: "wrapperPYth"

File: docs/constitution/pyth-dao-constitution.md [hunk 8]
```diff
@@ -2,46 +2,45 @@
...
+The Pyth DAO LLC, is legally structured as “a non-profit DAO LLC formed under the laws of the Republic of Marshall Islands, formed to serve the Pyth DAO LLC” (of which the OPERATING AGREEMENT OF PYTH DAO LLC is available at: [ipfs://QmP2GmL1n2WbHd7AtHqyXVWFyyHH36aZLfVZbNoqhommJi](https://cloudflare-ipfs.com/ipfs/QmP2GmL1n2WbHd7AtHqyXVWFyyHH36aZLfVZbNoqhommJi)). The wrapper Pyth DAO is algorithmically managed, such that actions taken by the Pyth DAO via the governance contract on the Solana Blockchain are deemed to be actions of the Pyth DAO LLC. The legal entity enables the Pyth DAO LLC to hold the treasury, protect Pyth DAO members from unlimited liability, and allow Pyth DAO members to take part in governance by providing a clear framework in respect of the rights and duties of Pyth tokenholders that hold Votable Tokens.
```
Comment (jayantk) [inline docs/constitution/pyth-dao-constitution.md:21]: suggest changing "wrapper Pyth DAO" to "Pyth DAO LLC"

Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: fixed

**Review by jayantk — APPROVED** [review]

**Pull request merged**